gnome-image-installer/pages/install/gis-install-page.c

```diff
@@ -285,8 +285,11 @@ gis_install_page_open_for_restore_cb (GObject      *source,
   gint fd = -1;
   g_autoptr(GError) error = NULL;
   g_autoptr(GFile) image = NULL;
+  const gchar *signature_path = NULL;
   g_autoptr(GFile) signature = NULL;
   g_autoptr(GisScribe) scribe = NULL;
+  guint64 uncompressed_size = gis_store_get_required_size ();
+  guint64 compressed_size = gis_store_get_image_size ();
```

Comment:
Nitpick: I’d put `_bytes` as a suffix on these variables names to make it clearer what units we’re working in.

Reply:
OK.